docs/docs/rest-api/public/api/v2/types/check.raml

```diff
@@ -0,0 +1,141 @@
+#%RAML 1.0 Library
+uses:
+  strings: stringTypes.raml
+  command: mesosCommand.raml
+  numbers: numberTypes.raml
+types:
+  HttpCheck:
+    type: object
+    properties:
+      endpoint:
+        type: strings.Name
+        description: |
+          The endpoint name to use.
+          In "host" mode checks use the hostPort. In other modes use the containerPort.
+      path?: strings.Path
+      scheme?:
+        type: strings.HttpScheme
+        default: HTTP
+  TcpCheck:
```

Comment:
Is this a `PodTcpCheck`?

Reply:
it is frustrating...  I was looking for pods and apps to be the same here..  it isn't possible.     There are generic TcpCheck and HttpChecks which it would be nice to use everywhere... however Apps to not have "endpoints" defined... pods do.  and we could go on and on.